gradle.properties

```diff
@@ -4,24 +4,24 @@ pluginGroup = intellij
 pluginName = intellij-buf
 pluginRepositoryUrl = https://github.com/bufbuild/intellij-buf
 # SemVer format -> https://semver.org
-pluginVersion = 0.4.3
+pluginVersion = 0.5.0
 
 # Supported build number ranges and IntelliJ Platform versions -> https://plugins.jetbrains.com/docs/intellij/build-number-ranges.html
-pluginSinceBuild = 223
-pluginUntilBuild = 243.*
+pluginSinceBuild = 243
```

Comment:
Is the idea that we don't want to support older versions at all? If the plugin still works under v2022, seems like we should keep it as is. But I guess maybe the issue is _testing_ with older versions?

Or is the issue that the code changes for 2024 won't work in earlier versions? I guess if we publish with this sort of metadata, does their plugin registry automatically handle providing users a version that will work with their version of IntelliJ?